### PR TITLE
Block edges from 4.7.z to 4.8.{2-4}

### PR DESCRIPTION
**Pull request by cblecker**
Blocking 4.7.z to 4.8.{2-4} edges due to BZ1988576.

### CODE REVIEW

**Comment by sdodson** [comment]
/hold
Awaiting confirmation that upgrading to 4.8.5 heals affected clusters or not.

File: blocked-edges/4.8.0.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+to: 4.8.0
+from: 4\.7\..*
+# Authentication operator fails to become available during upgrade https://bugzilla.redhat.com/show_bug.cgi?id=1988576
```
Comment (wking) [inline blocked-edges/4.8.0.yaml:3]: 4.8.0 was never supported (#918):

```console
$ curl -s 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=fast-4.8' | jq -r '.nodes[].version' | sort -V | tail -n5
4.7.23
4.8.2
4.8.3
4.8.4
4.8.5
```

I don't think we block updates for unsupported targets?

/hold

Comment (wking) [reply blocked-edges/4.8.0.yaml]: Fixed in 2297eae.

/hold cancel
/lgtm

**Pull request merged**